README.md

```diff
@@ -28,7 +39,7 @@ What is the purpose of the `alt` attribute? e.g:
 ```
 
 ```text
-Your answer here.
+The alt attribute assigns an alternative text to an image if the page cannot be displayed in the browser.
 ```
```

Comment:
:+1:

Also helpful for the vision-impaired. The content of the `alt` attribute is read out to them.